Cookbook/Technical-Documents/LabelsInPRs.md

```diff
@@ -0,0 +1,118 @@
+# How to label a Pull Request
+
+
+We use labels in our Pull Requests to be able to categorize and prioritize our work. Some specific labels are also used to trigger our CI jobs.
+
+
+## 1. The labels
+
+At the moment we have 26 labels in our repository. They can be separated into 4 groups:
+
+- [Product verticals](#product-verticals)
+
+- [Project horizontals](#project-horizontals)
+
+- [Branding](#branding)
+
+- [Status](#status)
+
+- [Emergency](#emergency)
+
+- [Extra](#extra)
+
+
+### <a name="product-verticals"></a>Product verticals
```

Comment:
[tip] you don't need to explicitly create HTML anchors with `<a name>`: The rendered markdown automatically creates HTML anchors for every section title (the anchor is the title, lowercased and with dashes in place of spaces)

Reply:
Good to know 😄🤦‍♀

Reply:
This doesn't seem to work if I remove it 😟

Reply:
This does seems to work for me though.

e,g, if I go back to your cdf974958732698b7cb7bc986211655bad22580a commit and ask GitHub to show me the rendered file, I can [jump to `#product-verticals` anchor](https://github.com/Babylonpartners/ios-playbook/blob/cdf974958732698b7cb7bc986211655bad22580a/Cookbook/Technical-Documents/LabelsInPRs.md#product-verticals) even though you removed the `a name` tags in that commit 🤔